### ISSUE TITLE
Mss screen grab makes screen responsiveness laggy (while gaming/moving a windows explorer)

### ISSUE THREAD

**Issue by vision34** [BoboTiG/python-mss#71]
General informations:

What i have done? Own implementation diy ambilight using mss screengrab

* OS name: Windows
* OS version: 10
* OS architecture: _64 bits_
* Resolutions (tested):
    * Monitor 1: 1080p@120Hz
* Python version: _2.7
* MSS version: last from pip
* CPU: i7 4790k @ 4.6Ghz
* GPU: GTX 1080 
### Description of the warning/error
Threaded screen capture is very laggy while gaming

### Full message

When i create a threaded class to capture the screen as fast as possible, it is makes my screen resposiveness very laggy... I can not play well. After making a sleep : Time.sleep(0.5), it does not slow down gaming, but reaction time for leds is too slow...

Pseudo code:

class Ambilight(Thread):
def __init__(self, LightController):
  self.screen = mss()
  self.imageCaptured = None
  self.LightController = LightController
...
...
def run(self):
 while True:
    self.imageCaptured = np.array(self.self.screen.grab(self.monitor[0]))
    self.LightController.ImageUpdate( self.imageCaptured.copy() )

**Comment by BoboTiG** [comment]
Hello,

If you remove all stuff non related to MSS, do you still see the lag?

**Comment by vision34** [comment]
Yes (i have used webcam as the source and it works fluently), it looks like when screen is captured, it freeze display for a little time...

**Comment by vision34** [comment]
https://github.com/vision34/AmbiPy/blob/master/AmbientLedProcessor.py here is my simple code

**Comment by Seraphli** [comment]
@vision34 I seem to experience the same issue. When I resize windows or drag windows or play games, the animation seems laggy. I'm also using gtx 1080, and capturing the screenshot every 0.4 s.
